Source/FullScreenPokemon.ts

```diff
@@ -4888,6 +4888,55 @@ module FullScreenPokemon {
             player.FSP.killNormal(player.bordering[player.direction]);
         }
 
+        /**
+         * Makes a StrengthBoulder move.
+         *
+         * @param player   The Player.
+         * @param pokemon   The Pokemon using Strength.
+         * @todo Eventually add check to make sure the Player beat the Gym leader needed to use the move.
+         * @todo Add an animation for what happens when the StrengthBoulder is moved.
+         */
+        partyActivateStrength(player: IPlayer, pokemon: IPokemon): void {
+            player.FSP.MenuGrapher.registerB();
+            player.FSP.MenuGrapher.registerB();
+            player.FSP.closePauseMenu();
+            player.FSP.TimeHandler.addEventInterval(
+                function (): void {
+                    var boulder: IHMSolid = <IHMSolid>player.bordering[player.direction];
+
+                    switch (player.direction) {
+                        case 0:
+                            boulder.xvel = 0;
```

Comment:
You're setting `.xvel` and `.yvel` but that doesn't do anything. It would be better to store them as variables outside the function, which means you wouldn't have the odd side effect of changing member properties of the boulder unnecessarily.